website/docs/r/spot_fleet_request.html.markdown

```diff
@@ -78,6 +78,91 @@ resource "aws_spot_fleet_request" "foo" {
 }
 ```
 
+### Example with a dynamic block
+
+```hcl
+resource "aws_spot_fleet_request" "cheap_compute" {
+  iam_fleet_role                      = "${aws_iam_role.aws-ec2-spot-fleet-tagging-role.arn}"
+  target_capacity                     = var.target_capacity
+  valid_until                         = "${timeadd(timestamp(), var.timeadd)}"
+  allocation_strategy                 = var.allocation_strategy
+  fleet_type                          = var.fleet_type
+  wait_for_fulfillment                = true
+  terminate_instances_with_expiration = true
+
+  launch_specification {
+    instance_type            = "m4.10xlarge"
+    ami                      = "ami-1234"
+    spot_price               = "2.793"
+    placement_tenancy        = "dedicated"
+    iam_instance_profile_arn = "${aws_iam_instance_profile.example.arn}"
+  }
+
+  launch_specification {
```

Comment:
This should be `dynamic "launch_specification"`.

Reply:
good catch... no idea how i missed that...